app/lib/seeder.rb

```diff
@@ -0,0 +1,39 @@
+# rubocop:disable Rails/Output
+
+class Seeder
```

Comment:
This was in seeds.rb, but I need it as well in e2e seeding, so I pulled it out to be shared.